corehq/project_limits/tests/test_rate_limiter.py

```diff
@@ -16,3 +18,39 @@ def test_rate_limit_interface():
     if my_feature_rate_limiter.allow_usage('my_domain'):
         # ...do stuff...
         my_feature_rate_limiter.report_usage('my_domain')
+
+
+@mock.patch('corehq.project_limits.rate_limiter.get_n_users_for_rate_limiting', lambda domain: 10)
+def test_get_window_of_first_exceeded_limit():
+    per_user_rate_def = RateDefinition(per_second=10)
+    min_rate_def = RateDefinition(per_second=100)
+    per_user_rate_def = PerUserRateDefinition(per_user_rate_def, min_rate_def)
+    rate_limiter = RateLimiter('my_feature', per_user_rate_def.get_rate_limits)
+    rate_limiter.iter_rates = Mock(return_value=[('second', 11, 10)])
+    expected_window = 'second'
+    actual_window = rate_limiter.get_window_of_first_exceeded_limit('my_domain')
+    eq(expected_window, actual_window)
```

Comment:
nit: I prefer and use opposite order for assert equal arguments: `eq(actual, expected)`. I think it reads more naturally. For example:

most people write: \<variable> == \<constant>
```py
if number == 3:
```

less common: \<constant> == \<variable>
```py
if 3 == x:
```

Both are valid code, of course.

In this case I think of the "actual" value more like a variable (it may vary if a bug causes the test to fail) while the "expected" value is more like a constant (it's normally fixed for a given test).

Also note that different test frameworks have different conventions. jUnit has `assertEquals(expected, actual)` while TestNG has `assertEquals(actual, expected)` [source](https://stackoverflow.com/a/26102965/10840). I'm not sure if our codebase is consistent throughout, and consistency is very important, so always prefer to match nearby code over global convention.